continuous_integration/travis_script.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+set -ex
+
+for f in spyderlib/*.py; do
+  if [[ $f == "spyderlib/restart_app.py" ]]; then
+    continue
+  fi
+  if [[ $f == "spyderlib/spyder.py" ]]; then
+    continue
+  fi
+  if [[ $f == "spyderlib/tour.py" ]]; then
+    continue
+  fi
+  if [[ $f == "spyderlib/start_app.py" ]]; then
+    continue
+  fi
+  if [[ $f == "spyderlib/pil_patch.py" ]]; then
+    continue
+  fi
+  python "$f"
+  if [ $? -ne 0 ]; then
+    exit 1
+  fi
+done
+
+
+for f in spyderlib/*/*.py; do
+  if [[ $f == spyderlib/plugins/*.py ]]; then
+    continue
+  fi
+  if [[ $f == spyderlib/qt/*.py ]]; then
+    continue
+  fi
+  if [[ $f == spyderlib/utils/environ.py ]]; then
+    continue
+  fi
+  if [[ $f == spyderlib/utils/qthelpers.py ]]; then
+    continue
+  fi
+  if [[ $f == spyderlib/utils/windows.py ]]; then
+    continue
+  fi
+  if [[ $f == spyderlib/widgets/*.py ]]; then
+    continue
+  fi
```

Comment:
If you are skipping `widgets` and `plugins`, what are exactly testing here?

Sorry, I don't get it :-)